test/sqllogictest/aggregates.slt

```diff
@@ -84,3 +84,67 @@ query RRRRRR
 SELECT variance(a), var_samp(a), var_pop(a), stddev(a), stddev_samp(a), stddev_pop(a) FROM t2
 ----
 0.9166666666666666  0.9166666666666666  0.6875  0.9574271077563381  0.9574271077563381  0.82915619758885
+
+# TODO(benesch): these filter tests are copied from cockroach/aggregate.slt;
```

Comment:
Is there an issue tracking aggregates that we can include the issue number in here for future grepping?

Reply:
Nope. There are like 7 different things wrong with cockroach/aggregate.slt at the moment, I'm afraid.